inst/unitTests/runit.eval.R

```diff
@@ -145,14 +145,20 @@ test.redirection <- function(){
     checkTrue(identical(capture.output(dummy <- .CallOctave('printf', out, buffer.std = -1)), out), "Octave output is still captured if stdout is not buffered")
     
     # Errors
-    oerr <- 'This is an Octave error indeed!!!'
-    checkException(.CallOctave('error', oerr), "R error is raised by Octave error")
+    #oerr <- 'This is an Octave error indeed!!!'
+
+    oerr <- "Error : RcppOctave::o_get - Could not find an Octave object named 'aaa'.\n       Match(es): aaaa\n\n"
+    #print('======================================================================')
+    #print(geterrmessage())
+    #print('======================================================================')
     checkTrue(grepl(oerr, geterrmessage(), fixed = TRUE), "Octave error message is correctly passed to R")
     checkException(.CallOctave('error', oerr), "R error is raised by Octave error (buffer.std = 2)")
-    checkTrue(grepl(oerr, geterrmessage(), fixed = TRUE), "Octave error message is correctly passed to R (buffer.std = 2)")
+    oerr2 <- 'Error in .CallOctave(\"error\", oerr) : \n  RcppOctave - Octave error: execution_exception\n'
```

Comment:
As in this case, execution_exceptions may not contain any additional info text. The wording of this error message is a bit cryptic. Maybe get rid of mentioning the C++ name of the exception and explain
"... RcppOctave - Octave error occurred while calling an octave function" with extra info added, if available?